recipes/qt/5.x.x/conanfile.py

```diff
@@ -730,37 +744,299 @@ def package_id(self):
                 self.info.settings.compiler.runtime = "MT/MTd"
 
     def package_info(self):
-        # FIXME add components
-        self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.names["cmake_find_package"] = "Qt5"
+        self.cpp_info.names["cmake_find_package_multi"] = "Qt5"
 
-        # Add top level include directory, so code compile if someone uses
-        # includes with prefixes (e.g. "#include <QtCore/QString>")
-        self.cpp_info.includedirs = ["include"]
+        libsuffix = ""
+        if self.settings.build_type == "Debug":
+            if self.settings.os == "Windows":
+                libsuffix = "d"
+            if tools.is_apple_os(self.settings.os):
```

Comment:
```suggestion
            else if tools.is_apple_os(self.settings.os):
```

Reply:
```suggestion
            elif tools.is_apple_os(self.settings.os):
```
If this is changed, I'd rather go for this !